controllers/nmstate_controller_test.go

```diff
@@ -31,7 +33,12 @@ var _ = Describe("NMState controller reconcile", func() {
 		reconciler          NMStateReconciler
 		existingNMStateName = "nmstate"
 		handlerNodeSelector = map[string]string{"selector_1": "value_1", "selector_2": "value_2"}
-		nmstate             = nmstatev1beta1.NMState{
+		handlerTolerations  = []corev1.Toleration{
+			{Effect: "NoSchedule",
```

Comment:
Indenting could be improved so that Effect, Key and Operator have all the same indent level:
```
    {
        Effect: ...,
        Key: ...,
        Operator: ...,
    }
````

Reply:
Ack'd. Fixed in latest commits.